packages/@react-aria/menu/src/useMenu.ts

```diff
@@ -72,7 +75,7 @@ export function useMenu<T>(props: AriaMenuOptions<T>, state: TreeState<T>, ref:
   });
 
   return {
-    menuProps: mergeProps(domProps, {
+    menuProps: mergeProps(domProps, {onKeyDown}, {
       role: 'menu',
       // this forces AT to move their cursors into any open sub dialogs, the sub dialogs contain hidden close buttons in order to come back to this level of the menu
       'aria-hidden': state.expandedKeys.size > 0 ? true : undefined,
```

Comment:
One thing that might become necessary is having `useMenu` accept the `menuTreeState` created by `useMenuState` and its own `level` so that any given menu/submenu check if it has a expandedKey for the above. Alternatively, we would need to move `useSubMenuTrigger` or something up a level so that it can initialize the `level` for a given Menu and thus setup an `expandedKeys` set to pass to useTreeState

Reply:
For now I've made it so we update each menu/submenus's treestate expanded keys when a submenu is expanded and collapsed for the above. It felt like the easiest path forward since we already have access to the TreeState here. Open for discussion

Reply:
As discussed in the meeting, we will be getting rid of this line since it is a screenreader edge case. Alternative may be to use the level from SubMenuTriggerState to look up if a menu has a expanded submenu instead of using `expandedKeys` to track